### ISSUE TITLE
'High Contrast White' theme button active state color

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1028]
**Describe the bug**
Button's active(pressed) state text color and the background color is the same: black

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/button.html
2. Click on any non emphasized button

**Expected behavior**
The button's active/pressed state text color should be white.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81854228-77cb8d00-9512-11ea-9019-8ae568c98059.png)

![image](https://user-images.githubusercontent.com/43764832/81854568-f58f9880-9512-11ea-949d-25236eb14cf4.png)

Similarly for segmented button:
![image](https://user-images.githubusercontent.com/43764832/81854475-d4c74300-9512-11ea-9b32-9a789c415e87.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 5:1
<img width="186" alt="Screen Shot 2020-10-07 at 5 44 07 PM" src="https://user-images.githubusercontent.com/21978807/95529934-3f30c200-0991-11eb-8c88-d47eed63a96c.png">
